src/pages/session/components/workItemView.tsx

```diff
@@ -142,6 +143,22 @@ class WorkItemView extends React.Component<IWorkItemProps & typeof Actions> {
                                                     );
                                                 })}
                                             </div>
+                                            {confidence && (
```

Comment:
I'm thinking of showing the average for all Numeric cards, there have been many asks for it. What do you think?

Reply:
seems good, what do you think @royds4 ?

Reply:
@cschleiden @amaury-biquet Agreed. Already updated.